### ISSUE TITLE
Auto shutdown delay?

### ISSUE THREAD

**Issue by enz1ey** [jamesmccannon02/OctoPrint-Tplinkautoshutdown#73]
Just wanted to make a suggestion, I think adding a customizable/optional delay for auto shutdown would be useful. I have some cooling fans connected to my printer's PSU with a relay so they are triggered when a print completes and run for ten minutes to cool the bed down quicker. Obviously, when my TPLink plug is turned off upon print completion, the fan shuts off as well.

**Comment by leftyfb** [comment]
No, really. This needs to be a thing. My Prusa definitely does not like being aggressively shut down immediately when the tray is trying to set itself and fans aren't even spun down yet. I'm trying to do this myself right after "if event == "PrintDone":" with a sleep() but I'm not very good at python.
